pkg/apis/rollouts/v1alpha1/analysis_types.go

```diff
@@ -316,6 +331,22 @@ type AnalysisRunStatus struct {
 	MetricResults []MetricResult `json:"metricResults,omitempty" protobuf:"bytes,3,rep,name=metricResults"`
 	// StartedAt indicates when the analysisRun first started
 	StartedAt *metav1.Time `json:"startedAt,omitempty" protobuf:"bytes,4,opt,name=startedAt"`
+	// DryRunSummary contains the final status of the metric results collected during the Dry-Run mode
+	DryRunSummary DryRunSummary `json:"dryRunSummary,omitempty" protobuf:"bytes,5,opt,name=dryRunSummary"`
+}
+
+// DryRunSummary contains the final results from the metric executions in the Dry-Run mode.
+type DryRunSummary struct {
+	// This is equal to the sum of Successful, Failed, Inconclusive
+	Count int32 `json:"count,omitempty" protobuf:"varint,1,opt,name=count"`
+	// Successful is the number of times the metric was measured Successful
+	Successful int32 `json:"successful,omitempty" protobuf:"varint,2,opt,name=successful"`
+	// Failed is the number of times the metric was measured Failed
+	Failed int32 `json:"failed,omitempty" protobuf:"varint,3,opt,name=failed"`
+	// Inconclusive is the number of times the metric was measured Inconclusive
+	Inconclusive int32 `json:"inconclusive,omitempty" protobuf:"varint,4,opt,name=inconclusive"`
+	// Error is the number of times an error was encountered during measurement
+	Error int32 `json:"error,omitempty" protobuf:"varint,5,opt,name=error"`
```

Comment:
This information appears to be aggregate information from the metric result list, but just provided as a convenience for humans? If so, I think it is better to remove this since:

1) I don't think we should have two sources of truth in the status. This information can be recaluclated from `status.metricResults` if necessary
2) the information is specific to dry run, so even if we were to provide summary information, I would think it should be for both dry runs and wet runs.

Reply:
As discussed offline, I have changed it to a `summary` instead and `dry-run summary` is part of this.